_data/speakers.yml

```diff
@@ -63,19 +63,20 @@
   subtitle: Speaker
   twitter: MoAkhshik
   github: makhshik
-  web:
+  web: 
   image: mo-akhshik.jpg
   thumbnail: mo-akhshik.jpg
   alt: Mo
-  topic:
-  proposal_summary:
+  topic: A Novel Bayesian Model to Fit Spectrophotometric Data of Hubble and Spitzer Space Telescopes
+  proposal_summary: Understanding how the most massive galaxies rapidly formed and quenched when Universe was only ~3 billion years old is one of the major challenges of extragalactic astronomy. In this talk, I will discuss how to improve our understanding of massive galaxy formation by combining the spectro-photometric observations of the Hubble and Spitzer Space Telescopes for strong gravitationally lensed galaxies. In particular, a multi-level regression model is built that can fit all multi-wavelength data for a range of instruments within a hierarchical Bayesian framework to constrain the properties of the stellar populations. The details of how this model is implemented using PyMC3, as well as the estimates of the posteriors of all parameters of interest and nuisance parameters will be highlighted.
   discourse_link:
   youtube_link:
   description: Mo is a grad student of (astro)physics by day, a matheux and a Bayesian enthusiast all along.
     Broadly interested in cosmology and probability too.
   presentation_type: Talk
-  time_zone:
+  time_zone: Africa/Asia/Europe
   track: Advanced
+  keywork: mohik
```

Comment:
```suggestion
  keyword: mohik
```

Reply:
Hey Hector, I had done that on purpose actually. Should I change them all?